fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -84,13 +85,13 @@ private void export(final URI uri) {
             if (linkHeaders.contains(binaryURI)) {
                 exportBinary(uri);
             } else if (linkHeaders.contains(containerURI)) {
-                exportDescription(uri);
+                exportDescription(uri, uri);
```

Comment:
No, what happened there is that I sent the wrong commit:(  I flipped that one parameter to test my integration test.  I'll fix it...